source/OptimaJet.DWKit.StarterApplication/Repositories/OrganizationInviteRequestRepository.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading.Tasks;
+using Hangfire;
+using JsonApiDotNetCore.Data;
+using JsonApiDotNetCore.Services;
+using Microsoft.Extensions.Logging;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Services;
+using Serilog;
+
+namespace OptimaJet.DWKit.StarterApplication.Repositories
+{
+	public class OrganizationInviteRequestRepository : DefaultEntityRepository<OrganizationInviteRequest>
+    {
+        public OrganizationInviteRequestRepository(
+            ILoggerFactory loggerFactory,
+            IJsonApiContext jsonApiContext,
+            IDbContextResolver contextResolver
+            ) : base(loggerFactory, jsonApiContext, contextResolver)
+        {
+            Log.Information("Constructed OrganizationInviteRequestRepository");
+        }
+
+        public override async Task<OrganizationInviteRequest> CreateAsync(OrganizationInviteRequest entity)
+        {
+            Log.Information("OrganizationInviteRequestRepository::CreateAsync");
```

Comment:
is this log important?

Reply:
Nope.  Needs to be removed.  It was there for testing.